modules/kargoBidAdapter.js

```diff
@@ -263,28 +264,41 @@ function getUserSyncs(syncOptions, _, gdprConsent, usPrivacy, gppConsent) {
   const seed = _generateRandomUUID();
   const clientId = getClientId();
 
-  var gdpr = (gdprConsent && gdprConsent.gdprApplies) ? 1 : 0;
-  var gdprConsentString = (gdprConsent && gdprConsent.consentString) ? gdprConsent.consentString : '';
-
-  var gppString = (gppConsent && gppConsent.consentString) ? gppConsent.consentString : '';
-  var gppApplicableSections = (gppConsent && gppConsent.applicableSections && Array.isArray(gppConsent.applicableSections)) ? gppConsent.applicableSections.join(',') : '';
+  const gdpr = gdprConsent?.gdprApplies ? 1 : 0;
+  const gdprConsentString = gdprConsent?.consentString || '';
+  const gppString = gppConsent?.consentString || '';
+  const gppApplicableSections = Array.isArray(gppConsent?.applicableSections) ? gppConsent.applicableSections.join(',') : '';
 
   // don't sync if opted out via usPrivacy
   if (typeof usPrivacy == 'string' && usPrivacy.length == 4 && usPrivacy[0] == 1 && usPrivacy[2] == 'Y') {
     return syncs;
   }
-  if (syncOptions.iframeEnabled && seed && clientId) {
-    syncs.push({
-      type: 'iframe',
-      url: CERBERUS.SYNC_URL.replace('{UUID}', clientId)
-        .replace('{SEED}', seed)
-        .replace('{GDPR}', gdpr)
-        .replace('{GDPR_CONSENT}', gdprConsentString)
-        .replace('{US_PRIVACY}', usPrivacy || '')
-        .replace('{GPP_STRING}', gppString)
-        .replace('{GPP_SID}', gppApplicableSections)
-    })
+
+  const createUrl = (url) =>
+    url.replace('{UUID}', clientId)
+      .replace('{SEED}', seed)
+      .replace('{GDPR}', gdpr)
+      .replace('{GDPR_CONSENT}', gdprConsentString)
+      .replace('{US_PRIVACY}', usPrivacy || '')
+      .replace('{GPP_STRING}', gppString)
+      .replace('{GPP_SID}', gppApplicableSections);
+
+  if (seed && clientId) {
+    if (syncOptions.iframeEnabled) {
+      syncs.push({
+        type: 'iframe',
+        url: createUrl(CERBERUS.SYNC_URL)
+      });
+    } else if (syncOptions.pixelEnabled) {
+      for (let i = 0; i < CERBERUS.SYNC_COUNT; i++) {
+        syncs.push({
+          type: 'image',
+          url: createUrl(`${CERBERUS.IMG_SYNC_URL}&idx=${i}`)
```

Comment:
passing `idx` 0 thru 4, but don't think it'll really be necessary to use on the backend since the sync selection process is random and probably doesnt need to be more complicated than that since the volume of sync opportunities is so great (ie., we've not implemented a 'once per user per day' approach, and probably don't need to).